internal/controller/ols_console_reconciliator.go

```diff
@@ -52,25 +49,29 @@ func (r *OLSConfigReconciler) reconcileConsoleUI(ctx context.Context, olsconfig
 			Name: "reconcile Console Plugin NetworkPolicy",
 			Task: r.reconcileConsoleNetworkPolicy,
 		},
+		{
+			Name: "reconcile Console Plugin Deployment",
+			Task: r.reconcileConsoleUIDeployment,
+		},
```

Comment:
@blublinsky Similar question , why do we need to change the reconcile order ?

Reply:
similar answer as above